lib/Doctrine/DBAL/Connection.php

```diff
@@ -435,6 +435,13 @@ private function getDatabasePlatformVersion()
             return $this->_conn->getServerVersion();
         }
 
+        if ($this->_conn instanceof \PDO && $this->_conn->getAttribute(PDO::ATTR_DRIVER_NAME)==='mysql') {
+            try {
+                return $this->_conn->query('select version()')->fetchColumn();
+            } catch (\PDOException $e) {
```

Comment:
I'd rather have a failure here, since it's mysql, and yet something is going horribly wrong. This `catch` block should go away.